Translators/PlannedEvents/pe_tim_generator.py

```diff
@@ -0,0 +1,10 @@
+from itis_codes import ItisCodes
+
+def get_itis_codes():
+    return [ItisCodes.CLOSED_FOR_SEASON.value]
+
+def get_geometry(geometry):
+    annotated_geometry = []
+    for coord in geometry:

```

Comment:
question: it looks like we're only getting two coordinate pairs for the closed roads. 
- Can we clarify with documentation or contacting Rob that this is expected and will be the start/end point of the road closure? 
- Are we handling the case for start/end point properly in the TIM Manager or is that expecting a linestring?

Reply:
I reached out to Rob to clarify this as it isn't included in the documentation, but haven't yet heard back. 
As for the start/end point handling, I had originally planned to handle this in the TIM Manager, however, I think it makes more sense to generate the correct geometry in the translator. There are a few translators that will use the geospatial service, so I'll handle that in a separate PR.